src/lib/packages.ts

```diff
@@ -456,7 +455,7 @@ export class TypingsVersions {
 
         this.map = new Map(this.versions.map(version => {
             const dataKey = versionMappings.get(version)!;
-            return [version, new TypingsData(data[dataKey], version === this.versions[0])];
+            return [version, new TypingsData(data[dataKey], version.equals(this.versions[0]))];
```

Comment:
what is the type of `version`? SemVer or string? It looks like this caught a real bug here.

Reply:
TS says the type is `Semver`. From what I understandd, this was working because luckily we were only comparing `Semver` objects that were part of the same array, so `===` and `.equals` have the same effect in this particular situation.